lms/djangoapps/oauth_dispatch/dot_overrides.py

```diff
@@ -0,0 +1,41 @@
+"""
+Classes that override default django-oauth-toolkit behavior
+"""
+
+from django.contrib.auth import authenticate as django_authenticate, get_user_model
+from oauth2_provider.oauth2_validators import OAuth2Validator
+
+
+def authenticate(username, password):
```

Comment:
Curious: why choose to make this a function, instead of a protected method on the `EdxOAuth2Validator` class?